lib/package.json

```diff
@@ -17,6 +17,7 @@
   "devDependencies": {
     "@babel/core": "^7.22.20",
     "@ledgerhq/wallet-api-client": "^1.5.0",
+    "@ledgerhq/wallet-api-exchange-module": "^0.1.1-custom-exchange.0",
```

Comment:
This is a strange version number :)

Reply:
I can update this after we publish a version from develop or main to npm